libs/dh/charges/feature-prices/src/components/information.component.ts

```diff
@@ -17,9 +17,125 @@
+              <h3>
+                @if (charge.validToDateTime && charge.validFromDateTime) {
+                  {{ dateRange(charge.validFromDateTime, charge.validToDateTime) | wattDate }}
+                } @else {
+                  {{ charge.validFromDateTime | wattDate }}
+                }
+                @if (
+                  charge.validToDateTime &&
+                  isCurrent(charge.validFromDateTime, charge.validToDateTime)
+                ) {
+                  <watt-badge type="success">{{ t('current') }}</watt-badge>
+                }
+              </h3>
+            </vater-stack>
+          </watt-card-title>
+          <watt-description-list variant="stack" [itemSeparators]="false">
+            <watt-description-list-item [label]="t('name')">
+              {{ charge.chargeName }}
+            </watt-description-list-item>
+            <watt-description-list-item [label]="t('description')">
+              {{ charge.chargeDescription }}
+            </watt-description-list-item>
+            <watt-description-list-item [label]="t('vatClassification')">
+              {{ 'charges.vatClassifications.' + charge.vatClassification | transloco }}
+            </watt-description-list-item>
+            <watt-description-list-item [label]="t('transparentInvoicing')">
+              {{ charge.transparentInvoicing ? ('yes' | transloco) : ('no' | transloco) }}
+            </watt-description-list-item>
+          </watt-description-list>
+        </watt-card>
+      }
+    </vater-stack>
+  `,
 })
-export class DhPriceInformationComponent {}
+export class DhPriceInformationComponent {
+  chargeInformations: Partial<ChargeInformationDto>[] = [
+    {
+      id: '2',
+      chargeName: 'Charge 2',
+      chargeDescription:
+        'En længere beskrivelse, der også kan fylde to linjer, eller måske endda endnu mere end to linjer.',
+      vatClassification: VatClassification.Vat25,
+      transparentInvoicing: false,
+      validFromDateTime: dayjs().subtract(1, 'year').toDate(),
+      validToDateTime: undefined,
+    },
+    {
+      id: '1',
+      chargeName: 'Charge 1',
+      chargeDescription:
+        'En længere beskrivelse, der også kan fylde to linjer, eller måske endda endnu mere end to linjer.',
+      vatClassification: VatClassification.NoVat,
+      transparentInvoicing: true,
+      validFromDateTime: dayjs().subtract(2, 'year').toDate(),
+      validToDateTime: dayjs().add(1, 'year').toDate(),
+    },
+    {
+      id: '3',
+      chargeName: 'Charge 3',
+      chargeDescription:
+        'En længere beskrivelse, der også kan fylde to linjer, eller måske endda endnu mere end to linjer.',
+      vatClassification: VatClassification.Vat25,
+      transparentInvoicing: false,
+      validFromDateTime: dayjs().subtract(2, 'year').toDate(),
+      validToDateTime: dayjs().subtract(1, 'year').toDate(),
+    },
+  ];
+
+  isCurrent(from?: Date, to?: Maybe<Date>) {
```

Comment:
nitpick: the `Maybe` type is `Maybe<T> = T | null` so the typing here is a bit too wide if `validToDateTime` can only be `Data | undefined.`